src/System.Threading/tests/System.Threading.Tests.csproj

```diff
@@ -45,6 +45,9 @@
       <Link>Common\System\IO\FileCleanupTestBase.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
```

Comment:
We are already by default testing in netcoreapp1.1, but do you want to add a build configuration in the tests.builds that passes in:
```xml
<TargetGroup>netstandard1.3</TargetGroup>
<TestTFMs>netcoreapp1.0</TestTFMs>
```